Zscaler/zscaler-zia/_meta/fields.yml

```
@@ -0,0 +1,84 @@
+zscaler.zia.event_id:
+  name: zscaler.zia.event_id
+  type: keyword
+  description: "ZScaler event ID"
+
+zscaler.zia.source_type:
+  name: zscaler.zia.source_type
+  type: keyword
+  description: "ZScaler source type"
+
+zscaler.zia.product:
+  name: zscaler.zia.product
+  type: keyword
+  description: "ZScaler product"
+
+zscaler.zia.vendor:
+  name: zscaler.zia.vendor
+  type: keyword
+  description: "ZScaler vendor"
+
+zscaler.zia.department:
+  name: zscaler.zia.department
+  type: keyword
+  description: "ZScaler department"
+
+zscaler.zia.category:
+  name: zscaler.zia.category
+  type: keyword
+  description: "ZScaler category"
+
+zscaler.zia.sub_category:
+  name: zscaler.zia.sub_category
+  type: keyword
+  description: "ZScaler sub category"
+
+zscaler.zia.action.result:
+  name: zscaler.zia.action.result
+  type: keyword
+  description: "ZScaler action result"
+
+zscaler.zia.audit.log_type:
+  name: zscaler.zia.audit.log_type
+  type: keyword
+  description: "ZScaler audit log type"
+
+zscaler.zia.threat.name:
+  name: zscaler.zia.threat.name
+  type: keyword
+  description: "ZScaler threat name"
+
+zscaler.zia.threat.class:
+  name: zscaler.zia.threat.class
+  type: keyword
+  description: "ZScaler threat class"
+
+zscaler.zia.threat.category:
+  name: zscaler.zia.threat.category
+  type: keyword
+  description: "ZScaler threat category"
+
+zscaler.zia.appname:
+  name: zscaler.zia.appname
+  type: keyword
+  description: "ZScaler app name"
+
+zscaler.zia.device.owner:
+  name: zscaler.zia.device.owner
+  type: keyword
+  description: "ZScaler device owner"
+
+zscaler.zia.keyprotectiontype:
+  name: zscaler.zia.keyprotectiontype
+  type: keyword
+  description: "ZScaler key protection type"
+
+zscaler.zia.tuntype:
+  name: zscaler.zia.tuntype
+  type: keyword
+  description: "ZScaler tunel type"
+
+zscaler.zia.avgduration:
+  name: zscaler.zia.avgduration
+  type: keyword
+  description: "ZScaler average duration"
```

With the declaration in the taxomony:
```suggestion
  description: "ZScaler average duration"

zscaler.zia.resource:
  name: zscaler.zia.resource
  type: keyword
  description: "ZScaler resource"
```